src/Presentation/Nop.Web/wwwroot/js/public.onepagecheckout.js

```diff
@@ -611,6 +623,25 @@ var ConfirmOrder = {
         }
     },
 
+    getCaptchaToken: async function (action) {
```

Comment:
I don’t quite understand why we need to get captcha token again here. It will already be on the page when button is clicked, we just need to find it and send it in the confirm request.

Reply:
on one page checkout time, nop generates the reCaptcha key with the action name OpcSavePaymentInfo as the confirm order part is loading from this method. So we are facing a problem when nop is going to validate the captcha. Because the post action name is 'OpcConfirmOrder'. For this methods name mismatch we need to generate the captcha with the action name 'OpcConfirmOrder'.